dataactvalidator/filestreaming/csvAbstractReader.py

```diff
@@ -207,7 +207,6 @@ def _getLine(self):
             self.packetCounter +=1
 
             #Get the current lines
-            packet = packet.decode("utf-8") if not self.isLocal else packet
             currentBytes = self.unprocessed + packet
```

Comment:
Sanity check: `currentBytes` is supposed to hold a unicode string, not a byte sequence, correct?